google/location.go

```diff
@@ -30,22 +30,21 @@ func (l *Location) CreateContainer(containerName string) (stow.Container, error)
 
 	projId, _ := l.config.Config(ConfigProjectId)
 	location, _ := l.config.Config(ConfigLocation)
-	storageClass, _ := l.config.Config(ConfigStorageClass)
 
 	// Create a bucket.
-	_, err := l.client.Buckets.Insert(projId, &storage.Bucket{
-		Name:         containerName,
-		Location:     location,
-		StorageClass: storageClass,
```

Comment:
Why removed storage class? They represent properties such as SLA, cost etc., which are different from location property, right?

Reply:
I will re-add it Tomorrow as an option

Reply:
Thanks for the clarification. Since stow is a standalone project, where possible, I prefer we retain its original configuration options, unless they are causing issues downstream.

Reply:
re-added